src/main/java/uk/gov/hmcts/reform/data/ingestion/DataIngestionLibraryRunner.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.data.ingestion;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.CamelContext;
+import org.springframework.batch.core.Job;
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.launch.JobLauncher;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.data.ingestion.camel.service.AuditServiceImpl;
+
+import java.util.Date;
+
+import static java.lang.Boolean.TRUE;
+import static uk.gov.hmcts.reform.data.ingestion.camel.util.MappingConstants.SCHEDULER_START_TIME;
+
```

Comment:
Please add java doc comments at class and method level

Reply:
done